### ISSUE TITLE
Questions about the experiment

### ISSUE THREAD

**Issue by La-SilverLand** [ashkamath/mdetr#4]
hi, i've got 2 questions

1) in the few-shot transfer learning experiment on LVIS, the 100%-data's performance is not better than the 10%-data's for MDETR over all metrics, why is that ? ( i only found the small object detection drop being mentioned)

2) for referring image segmentation task, have you tested on the UNC/UNC+/G-ref/ReferIt, these're frequently mentioned tasks for the referring image segmentation.  They're based on MS COCO, but i noticed that you excluded the val/test set of COCO in the pretraining

thanks

**Comment by ashkamath** [comment]
Hello, 

1. It was supposed to be an explanation for "rare" classes. That was a typo. But the reason is what we mentioned, that the class imbalance is much more amplified in the 100%. As you can see the AP frequent is better than DETR while AP rare is worse which supports our understanding that it is the class imbalance that's causing this. Further it might also be that our model is undertrained (we ran around 3 epochs on full) as opposed to the 10% model and plan to look into this in future. 
2. We did not run segmentation on these datasets, but based on the phrasecut results, we believe the scores would be similar to what we have reported on these datasets within a few points difference. If you are interested in trying it out, the approach would be identical to the instructions we have posted for phrasecut. Note: the train/val/test splits of referring expressions all come from COCO train, and we exclude these images from our pre-training dataset.
